lib/gtfs_realtime_viz.ex

```diff
@@ -167,15 +176,35 @@ defmodule GTFSRealtimeViz do
     Enum.sort(time_list, &time_list_sorter/2)
   end
 
-  defp time_list_sorter({_, time1}, {_, time2}) do
+  defp time_list_sorter({_, {arr_time1, dep_time1}}, {_, {arr_time2, dep_time2}}) do
+    time1 = arr_time1 || dep_time1
+    time2 = arr_time2 || dep_time2
     Timex.before?(time1, time2)
   end
 
   defp format_time({_, nil}) do
     nil
   end
-  defp format_time({trip_id, time}) do
-    ascii = Timex.format!(time, "{h24}:{m}:{s}")
+  defp format_time({trip_id, {arr_time, dep_time}}) do
+    now = Timex.now
+
+    ascii_arr = if arr_time do
+      diff = Float.round(Timex.diff(arr_time, now, :seconds) / 60, 1)
```

Comment:
is the decimal point valuable here?  on the one hand, its more precise and can help to give context on why the sign might differ from this if the rounded value is slightly different between here and the sign - on the other hand the sign always displays a round number and therefore maybe we should try to emulate that.

that said, i realize this isnt necessarily an mbta thing so maybe we can think about this more generally and maybe then more information is better.  We are the primary consumer at the moment though so its probably ok to think about it from our use case perspective.

Reply:
Yeah, it's a good question, and I agree that we can consider from our perspective. My thinking was that "rounding" is kind of an open question and the signs could handle it differently. Like, rounding vs flooring, when we switch to ARR or BRD, if we're smoothly counting down or kind of jumping around minute by minute. I still wanted to display it in minutes rather than seconds, since that _is_ what the signs display it in, but I thought a little bit more precision could help us debug issues. I'm not married to it, although I'd need to know precisely what rounding logic to put here.

Reply:
the one concern i have with the decimal is that it implies more accuracy than we are really giving - in an ideal world we would be showing the diff between the predicted time and the time on the sign, i think.  

For now unless something changes and we have a different/better idea of what we want, this seems good to me though.